### ISSUE TITLE
Issue with remove() function 

### ISSUE THREAD

**Issue by jigneshHchauhan** [MurhafSousli/ngx-gallery#405]
#### What is the expected behavior?

- Image should be removed from gallery and count should be updated.

#### What is the current behavior?

- Image is removed but count is not updated as well as it is showing black screen on place of removed image.

#### Which versions are you using for the following packages?
Angular: "12"
Angular CDK:  "12.0.0"
Angular CLI: "12.0.5"
Typescript: "4.2.3"
Gallery: "5.0.0"

![Screenshot (133)_LI](https://user-images.githubusercontent.com/59005409/135440189-882d008d-1530-47ce-9df0-c7e7208440d3.jpg)
![Screenshot (134)_LI](https://user-images.githubusercontent.com/59005409/135440210-ba16750e-f2ed-4c81-bcf5-1067eb53c0e1.jpg)

`   this.gallery.ref("myGallery").load([
      new ImageItem({ src: 'antarctica-iceberg-ocean.jpg', thumb: 'antarctica-iceberg-ocean.jpg' }),
      new ImageItem({ src: '1945822.jpg', thumb: '1945822.jpg' }),
      new ImageItem({ src: 'forest-mountains-sunset-river.jpg', thumb: 'forest-mountains-sunset-river.jpg' })
    ]); 
   
   this.gallery.ref("myGallery").remove(1); 
    `

**Comment by MurhafSousli** [comment]
Good catch! a workaround for now is just to set the the active element again!

```
remove(gallery: GalleryRef) {
    gallery.remove(2);
    gallery.set(2);
}
```

Here is a working [stackblitz](https://stackblitz.com/edit/ngx-gallery-wts5gl?file=src%2Fapp%2Fhome%2Fhome.component.ts)

**Comment by jigneshHchauhan** [comment]
The workaround is working fine for all the indexes but if we remove last image than the same problem occur😅

**Comment by shreyshahh** [comment]
@jigneshHchauhan In that case, you need to set index of last image - 1
```
remove(gallery: GalleryRef) {
    gallery.remove(3); // last image index
    gallery.set(2);
}
```

**Comment by MurhafSousli** [comment]
Fixed in v5.1.1
